### PR TITLE
chore(deps): Bump community-id from 0.2.2 to 0.2.3

### PR DESCRIPTION
**Pull request by DylanRJohnston**
Solves #21390

### CODE REVIEW

**Review by jszwedko — APPROVED** [review]
Thanks @DylanRJohnston !

**Comment by DylanRJohnston** [comment]
Hey @jszwedko it's not clear to me what is holding up the auto merge, it says it still requires approval from a maintainer?

**Comment by jszwedko** [comment]
> Hey @jszwedko it's not clear to me what is holding up the auto merge, it says it still requires approval from a maintainer?

Apologies, I just needed to approve CI. It should be going now.

**Pull request merged**
